_posts/2024-06-01-galois.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "Trapdoor author writeup"
```

Comment:
A common `GPNCTF:` prefix or category would be nice

Reply:
done